scyjava/__init__.py

```diff
@@ -15,19 +15,35 @@ def _init_jvm():
         import jnius
         return jnius
 
+    # attempt to find pyjnius.jar if the envrionment variable is not set.
     PYJNIUS_JAR_STR = 'PYJNIUS_JAR'
     if PYJNIUS_JAR_STR not in globals():
         PYJNIUS_JAR = None
         try:
-            PYJNIUS_JAR = Path(os.environ[PYJNIUS_JAR_STR])
+            PYJNIUS_JAR = os.environ[PYJNIUS_JAR_STR]
         except KeyError as e:
-            PYJNIUS_JAR = Path(sys.prefix) / 'share/pyjnius/pyjnius.jar'
-        if PYJNIUS_JAR.is_file():
+            PYJNIUS_JAR = sys.prefix + '/share/pyjnius/pyjnius.jar'
```

Comment:
While modifying this, we should probably change this (and fix this existing bug) to
```python
PYJNIUS_JAR = Path(sys.prefix) / 'share' / 'pyjnius' / 'pyjnius.jar'
```
or
```python
PYJNIUS_JAR = os.path.join(sys.prefix, 'share', 'pyjnius', 'pyjnius.jar')
```

Reply:
Aha, thanks @hanslovsky!